actionview/test/template/form_helper/form_with_test.rb

```diff
@@ -339,14 +339,6 @@ def url_for(object)
     super
   end
 
-  def test_form_with_when_given_nil_model_argument
```

Comment:
You should keep the test, but use `assert_deprecated` instead of `assert_raises`.

Reply:
I didn't know `assert_deprecated` was a thing (my first time using this deprecation stuff) so thanks for this one!